src/components/color-select/color-select.module.scss

```diff
@@ -0,0 +1,41 @@
+.root {
+    display: flex;
+    gap: 8px;
+}
+
+.option {
+    height: 20px;
+    width: 20px;
+    border: 1px solid var(--primary3);
```

Comment:
Let's do what we did for the slider component or this could be reused with no changes in the other template.